src/main/scala/cromwell/engine/backend/local/LocalEngineFunctions.scala

```diff
@@ -15,7 +15,9 @@ class LocalEngineFunctions(executionContext: TaskExecutionContext) extends Engin
   override protected def read_int(params: Seq[Try[WdlValue]]): Try[WdlInteger] = {
     assertSingleArgument(params).map {
```

Comment:
We should decide as a team (if not within broader DSDE) what we want this limit to be.  This is 110 characters long as 1 line, which is longer than I would like.